README.md

```diff
@@ -119,3 +119,54 @@ usage, but rather an auto-saved JSON file.
 MySQL database holds user, template, and project info. Web application intended 
 to be run on Apache Web Server. HTML/CSS/JS frontend with PHP api and backend 
 and MySQL DB
+
+
+## Testing
+The end-to-end tests in this project support both MySQL and SQLite databases.
+However, there are differences in table creation when using SQLite. 
+To facilitate SQLite database testing, a separate SQL file named `new-schema.sql` has been created under the `tests/sql folder.
+
+### Prerequisite
+
+Before running the tests, ensure that the following prerequisites are met:
+
+#### SQL Files:
+* The tests rely on the `docs/new-schema.sql` and `docs/seed.sql` files.
+* Any changes made to `docs/new-schema.sql` should also be incorporated into `tests/sql/new-schema.sql` to ensure proper SQLite database support.
+* The `docs/seed.sql` file contains test data used by the tests. If tests are failing due to changes in the data, update the relevant data inside the test file `tests/integration/UploadSlideBackTranslationIntegrationTest.php`.
+
+#### Test Story Template: 
+* Inside the `tests` directory, a test story template is located in the `tests/data/templates` directory.
+* If tests are failing  due to any changes in this directory, make the necessary adjustments in the test file `tests/integration/UploadSlideBackTranslationIntegrationTest.php`.
```

Comment:
Please remove the extra space between "failing" and "due".

Can you clarify what the adjustment is? Better yet, can the test setup do the adjustments?